.env

```
@@ -106,6 +106,7 @@ _APP_INTERVAL_CLEANUP_STALE_EXECUTIONS=300
 _APP_USAGE_STATS=enabled
 _APP_LOGGING_CONFIG=
 _APP_LOGGING_CONFIG_REALTIME=
+_APP_GRAPHQL_INTROSPECTION=enabled
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_GRAPHQL_INTROSPECTION key should go before the _APP_INTERVAL_CLEANUP_STALE_EXECUTIONS key

(UnorderedKey)